### PR TITLE
Refactoring tests

### PR DESCRIPTION
**Pull request by hsouf**

### CODE REVIEW

**Comment by hsouf** [comment]
few nitpicks

File: contracts/l2/mocks/mock_static_a_token.cairo [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.starknet.common.syscalls import get_caller_address
+from starkware.cairo.common.uint256 import Uint256
+from starkware.cairo.common.bool import TRUE
+from starkware.cairo.common.math import assert_not_zero
+
+from contracts.l2.dependencies.openzeppelin.token.erc20.library import ERC20
+
+from contracts.l2.dependencies.openzeppelin.access.ownable import Ownable
+
```
Comment (achab) [inline contracts/l2/mocks/mock_static_a_token.cairo:12]: this as well

File: pyproject.toml [hunk 2]
```diff
@@ -5,7 +5,7 @@ description = "A bridge of aave staticATokens for reward collection on L2"
 authors = ["AAVE", "nethermind"]
 
 [tool.poetry.dependencies]
-python = "3.7.12"
+python = "3.7.13"
```
Comment (achab) [inline pyproject.toml:8]: why do we need to upgrade python here?

File: test/bridge.test.ts [hunk 3]
```diff
@@ -32,10 +32,11 @@ chai.use(solidity);
 const MAX_UINT256 = hre.ethers.constants.MaxInt256;
 
 // Amount of dai and usdc to transfer to the user. Issued twice for aDai and aUsdc
-const DAI_UNIT = 1000n * BigInt(10 ** 18);
-const USDC_UNIT = 1000n * BigInt(10 ** 6);
+const DAI_UNIT = BigInt(10 ** 18);
```
Comment (achab) [inline test/bridge.test.ts:35]: nice, I've introduced these `1000n` when we used to have very small amounts of reward. good that it works even without it!

**Review by achab — APPROVED** [review]
Thanks a lot for this refactoring @hsouf ! Once comments are addressed, feel free to merge this PR

File: pyproject.toml (hunk 2 above)
Comment (hsouf) [reply pyproject.toml:8]: the ci suddenly refused to run with an older version see: https://github.com/aave-starknet-project/aave-starknet-bridge/runs/8204649309?check_suite_focus=true

Comment (achab) [reply pyproject.toml:8]: wow, really weird. it means that packages are not perfectly locked, no?

Comment (achab) [reply pyproject.toml:8]: anyway, ok to upgrade it if it works well with python 3.7.12. don't forget to also change README.md that mentions Python version to use

**Pull request merged**